meinberlin/apps/users/forms.py

```diff
@@ -112,10 +110,11 @@ def __init__(self, *args, **kwargs):
 
     def save(self, request):
         user = super(TermsSignupForm, self).save(request)
-        user.get_newsletters = self.cleaned_data["get_newsletters"]
-        user.get_notifications = self.cleaned_data["get_notifications"]
-        user.save()
-        return user
+        if user:
```

Comment:
the super().save() sends the email and returns None in case of email already exists.